typescript/cli/scripts/run-e2e-test.sh

```diff
@@ -3,16 +3,20 @@
 function cleanup() {
   set +e
   pkill -f anvil
+  rm -rf /tmp/anvil1
   rm -rf /tmp/anvil2
   rm -rf /tmp/anvil3
+  rm -f ./test-configs/anvil/chains/anvil1/addresses.yaml
   rm -f ./test-configs/anvil/chains/anvil2/addresses.yaml
   rm -f ./test-configs/anvil/chains/anvil3/addresses.yaml
   set -e
 }
 
 cleanup
 
-echo "Starting anvil2 and anvil3 chain for E2E tests"
+echo "Starting anvil1, anvil2 and anvil3 chain for E2E tests"
+# Anvil1 should be used only for core commands e2e testing to avoid interfierence with other e2e tests
```

Comment:
why are you unable to reuse anvil2/3?

also typo
```suggestion
# Anvil1 should be used only for core commands e2e testing to avoid interference with other e2e tests
```